package.json

```
@@ -1,13 +1,25 @@
 {
-  "name": "EngageSphere-Test-Design-Masterclass-turma-2",
+  "name": "engagesphere-test-design-masterclass",
   "version": "1.0.0",
   "description": "Sample project with a Node.js backend and a React frontend.",
   "scripts": {
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "lint": "eslint . --ignore-pattern 'backend/*' --ignore-pattern 'frontend/*'",
+    "api:run:cloud": "cypress run --record --tag 'API' --spec cypress/e2e/api-tests/**.cy.js",
```

```suggestion
    "api:run:cloud": "cypress run --record --tag 'API' --spec 'cypress/e2e/api-tests/**.cy.js'",
```

Ao passar uma expressão ao `--spec`, deixe-a entre aspas.

Isso evita um _warning_ do Cypress.

> O mesmo vale para outros lugares onde isso ocorre.